Kinetic_space_partition/include/CGAL/KSP_3/Initializer.h

```diff
@@ -499,7 +493,7 @@ class Initializer {
 
                 typename Intersection_graph::Kinetic_interval& kinetic_interval = m_data.igraph().kinetic_interval(e, sp_idx);
                 crossing_iedges.push_back(e);
-                if (emin > s) {
+                if (emin > s || (min_speed != min_speed)) {
```

Comment:
Is this for testing against e.g. NaN or ...?
In the first case it is probably clearer to use `!std::isNaN(min_speed)`

Reply:
By head I gave `isnan` as possibility, but maybe it is better to use `isfinite`, also checkout `isinf` and `isnormal`

Reply:
Thank you. isinf is actually the correct one.